jans-cedarling/bindings/cedarling_python/tests/config.py

```diff
@@ -6,11 +6,17 @@
 ROOT_FOLDER_PATH = "../../"
 TEST_FILES_PATH = join(ROOT_FOLDER_PATH, "cedarling/src/init/test_files")
 
+# in `sample_bootstrap_config` we use policy store `policy-store_ok.json`
+# The human-readable policy and schema file is located in next folder:
+# `../..cedarling/src/init/test_files/policy-store_ok`
+
 
-# we use fixture with correct data
-# and can change some part in the specific test case
 @pytest.fixture
 def sample_bootstrap_config():
+    '''
+    Fixture with correct data of BootstrapConfig.
+    You can change some part in the specific test case.
+    '''
     # log_config = MemoryLogConfig(log_ttl=100)
     # log_config = StdOutLogConfig()
```

Comment:
I think we should keep logging enabled by default when running the tests so that we can examine the decision, system logs when running authz.

Reply:
During execution unit tests with `pytest` framework, we can't see any logs.

Reply:
Also if we see tons of logs during execution pytest. It would be impossible to figure out to what exactly is going on.

Screen with enabled logger to stdout. Pytest blocks all output
![image](https://github.com/user-attachments/assets/671dd162-3567-43fb-8f95-abd851589ae1)

Reply:
> Also, if we see tons of logs during execution pytest. It would be impossible to figure out what exactly is going on.

The test cases will only print logs if `log_config = StdOutLogConfig()` otherwise not.

> During execution unit tests with pytest framework, we can't see any logs.

@SafinWasi it will be kind if you can comment on this point.

Reply:
Normally pytest does not display any program output to stdout, and this is by design. However, it is possible to configure pytest to capture these logs: https://docs.pytest.org/en/6.2.x/capture.html#setting-capturing-methods-or-disabling-capturing

Reply:
Resolved and added comment about command `pytest -s`